README.md

```diff
@@ -10,3 +10,19 @@
 ## Licence
 
 Apache 2
+
+## Releasing
+
+We don't use `zest.releaser` right now because of some problems with
+releasing a package that is not at the root of the repo (`jobs/`), and
+because we have no interest in uploading this project to PyPI, but
+this could change if we figure out how.
+
+The current procedure is:
+
+* Bump version in `jobs/setup.py`
+* `git tag x.y.z`
```

Comment:
What about updating the release date in the `CHANGELOG.rst` and running `make build-dependencies`